modules/nf-core/sourmash/sketch/meta.yml

```diff
@@ -1,15 +1,15 @@
 name: sourmash_sketch
-description: Create a signature (a hash sketch) of a sequence using sourmash
+description: Create a signature (a group of FracMinHash sketches) of a sequence using sourmash
 keywords:
   - hash sketch
   - signature
```

Comment:
```suggestion
  - sourmash
  - genomics
  - metagenomics
  - taxonomic classification
  - taxonomic profiling
  - kmer
```

Reply:
to sync with https://github.com/nf-core/modules/pull/2250/files, and make it more findable on https://nf-co.re/modules